patterns/creational/factory.py

```diff
@@ -29,27 +29,29 @@
 class GreekLocalizer:
     """A simple localizer a la gettext"""
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.translations = {"dog": "σκύλος", "cat": "γάτα"}
 
-    def localize(self, msg):
+    def localize(self, msg: str) -> str:
         """We'll punt if we don't have a translation"""
         return self.translations.get(msg, msg)
 
 
 class EnglishLocalizer:
     """Simply echoes the message"""
 
-    def localize(self, msg):
+    def localize(self, msg: str) -> str:
         return msg
 
 
-def get_localizer(language="English"):
+def get_localizer(language: str = "English",) -> object:
```

Comment:
Is the comma required here? Also, please check with a linter like `pep8`. AFAIR spaces around arguments are not recommended.

Reply:
Removed the trailing comma and checked with *black* formatter with default configuration.